client/src/plus/app.tsx

```diff
@@ -438,7 +452,16 @@ export default function App() {
               </>
             ) : (
               data &&
-              data.variant && <LandingPageSurvey variant={data.variant} />
```

Comment:
I don't know if it makes sense to hide the waitlist signup on desktop as well because it's not a pattern I've seen anywhere else.